java/java-analysis-impl/src/com/intellij/codeInspection/localCanBeFinal/LocalCanBeFinal.java

```diff
@@ -280,9 +365,7 @@ public void visitForStatement(@NotNull PsiForStatement statement) {
               }
               final PsiElement[] declaredElements = declarationStatement.getDeclaredElements();
               for (final PsiElement declaredElement : declaredElements) {
-                if (declaredElement instanceof PsiVariable) {
-                  result.add((PsiVariable)declaredElement);
-                }
+                if (declaredElement instanceof PsiVariable) result.add((PsiVariable)declaredElement);
```

Comment:
Please rollback the stylistical changes like this. The PR is too long already, and these complicate review. Not to mention that such a change is questionable one.